pkg/aws/aws_client/capacity_reservations.go

```diff
@@ -0,0 +1,46 @@
+package aws_client
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/openshift-online/ocm-common/pkg/log"
+)
+
+func (client *AWSClient) CreateCapacityReservation(crInput *ec2.CreateCapacityReservationInput, endDate time.Time, creator string) (*ec2.CreateCapacityReservationOutput, error) {
```

Comment:
Can you please elaborate on the need to have this in ocm-common? which projects are expected to use it?

Do you plan to add unit tests for this?

time.Now() does not seem to comply with UTC() - my concern is when run in multiple regions it may create confusion.

Reply:
this is going to be consumed by the fvt e2e test for now